tileserver/__init__.py

```diff
@@ -234,9 +234,10 @@ def async_store(store, tile_data, coord, format):
 def async_update_tiles_of_interest(redis_cache_index, coord):
     """update tiles of interest set"""
     try:
-        redis_cache_index.index_coord(coord)
+        if coord.zoom <= 20:
+            redis_cache_index.index_coord(coord)
         if coord.zoom > 18:
-            coord_at_z18 = coord.zoomTo(18)
+            coord_at_z18 = coord.zoomTo(18).container()
```

Comment:
Yup, exactly. We're actually not even storing request counts there. It's just a set that we perform lookups against to determine if we should process a particular tile.